examples/inverse/plot_morph_stc_to_fsaverage_dipy.py

```diff
@@ -0,0 +1,231 @@
+mne.save_stc_as_volume('lcmv_inverse.nii.gz', stc,
+                       forward['src'], mri_resolution=True)  # set to true
+
+###############################################################################
+# Load nifti data
+
+# load lcmv inverse
+stc = nib.load('lcmv_inverse.nii.gz')
+
+# select first time volume and output ndarray
+stc = stc.dataobj[:, :, :, 1]
+
+# normalize values
+stc = stc / stc.max()
+
+# load subject brain (Moving)
+t1_fname = data_path + '/subjects/sample/mri/brain.mgz'
+t1_m = nib.load(t1_fname)
+
+# get Moving to world transform
+t1_m_grid2world = t1_m.affine
+
+# output Moving as ndarray
+t1_m = t1_m.dataobj[:, :, :]
+
+# normalize values
+t1_m = t1_m.astype('float') / t1_m.max()
+
+# load fsaverage brain (Static)
+t1_fname = data_path + '/subjects/fsaverage/mri/brain.mgz'
+t1_s = nib.load(t1_fname)
+
+# get Static to world transform
+t1_s_grid2world = t1_s.affine
+
+# output Static as ndarray
+t1_s = t1_s.dataobj[:, :, :]
+
+# normalize values
+t1_s = t1_s.astype('float') / t1_s.max()
+
+##############################################################################
+# Compute affine transformation using mutual information metric
+
+# compute center of mass
+c_of_mass = transform_centers_of_mass(t1_s, t1_s_grid2world,
+                                      t1_m, t1_m_grid2world)
+
+nbins = 32
+
+# prepare affine registration
+affreg = AffineRegistration(metric=MutualInformationMetric(nbins, None),
+                            level_iters=[10000, 1000, 100],
+                            sigmas=[3.0, 1.0, 0.0],
+                            factors=[4, 2, 1])
+
+# translation
+translation = affreg.optimize(t1_s, t1_m, TranslationTransform3D(), None,
+                              t1_s_grid2world, t1_m_grid2world,
+                              starting_affine=c_of_mass.affine)
+
+# rigid body transform (translation + rotation)
+rigid = affreg.optimize(t1_s, t1_m, RigidTransform3D(), None,
+                        t1_s_grid2world, t1_m_grid2world,
+                        starting_affine=translation.affine)
+
+# affine transform (translation + rotation + scaling)
+affine = affreg.optimize(t1_s, t1_m, AffineTransform3D(), None,
+                         t1_s_grid2world, t1_m_grid2world,
+                         starting_affine=rigid.affine)
+
+# apply affine transformation
+t1_m_affine = affine.transform(t1_m)
+
+##############################################################################
+# Compute Symmetric Diffeomorphic Registration
+
+# set up Symmetric Diffeomorphic Registration (metric, iterations per level)
+sdr = SymmetricDiffeomorphicRegistration(CCMetric(3), [10, 10, 5])
+
+# compute mapping
+mapping = sdr.optimize(t1_s, t1_m_affine)
+
+##############################################################################
+# Apply transformations and plot
+
+# morph stc data
+stc_sdr_affine = mapping.transform(affine.transform(stc))
+
+# plot result
+slice_sel = 100
+
+fig = plt.figure()
```

Comment:
can we use a standard `nilearn` plotter instead? See `nilearn` docs for details.